app/views/job_runs/_structural_updates_summary.html.erb

```diff
@@ -0,0 +1,25 @@
+<div class="structural_changes">
+    <h5>Updates from csv</h5>
+    <table class="table">
+    <thead class="table">
```

Comment:
Did you find you needed this class on the `<thead>` element? I'm not familiar with it.

Reply:
I do believe it is required for a11y work, but I could be mistaken.